ReadMe followup for  #2313 (#2317)
paritytech/cumulus: README.md

chain node. By default, only the first URL will be used, with the rest as a backup in case the
connection to the first node is lost.

Parachain nodes using this feature won't have to fully sync with the relay chain to work, so in general
they will use fewer system resources.

**Note:** At this time, any parachain nodes using this feature will still spawn a significantly cut-down
relay chain node in-process. Even though they lack the majority of normal Polkadot subsystems, they
will still need to connect directly to the relay chain network.
##### Example command
```shell=
polkadot-parachain \
	--chain parachain-chainspec.json \
	--tmp \
	--relay-chain-rpc-urls \
		"ws://relaychain-rpc-endpoint:9944" \
		"ws://relaychain-rpc-endpoint-backup:9944" \
	-- \
	--chain relaychain-chainspec.json
```

## Installation and Setup
Before building Cumulus SDK based nodes / runtimes prepare your environment by following Substrate
[installation instructions](https://docs.substrate.io/main-docs/install/).

To launch a local network, you can use [zombienet](https://github.com/paritytech/zombienet) for
quick setup and experimentation or follow the [manual setup](#manual-setup).

### Zombienet

> We use Zombienet to spin up networks for integration tests and local networks.

Reviewer: ```suggestion
Zombienet is used to spin up networks for integration tests and local networks.
```

Follow [these installation steps](https://github.com/paritytech/zombienet#requirements-by-provider)

> to set it up on your machine. A simple network specification with two relay chain nodes and one collator is

Reviewer: ```suggestion
to set it up on local machine. A simple network specification with two relay chain nodes and one collator is
```
